fastn-js/js/dom.js

```diff
@@ -2112,7 +2112,7 @@ class Node2 {
         } else if (kind === fastn_dom.PropertyKind.Fit) {
             this.attachCss("object-fit", staticValue);
         } else if (kind === fastn_dom.PropertyKind.FetchPriority) {
-            this.attachAttribute("src", staticValue);
+            this.attachAttribute("fetch-priority", staticValue);
```

Comment:
This should be same as the HTMLImageElement property name, i.e. `fetchpriority`
```suggestion
            this.attachAttribute("fetchpriority", staticValue);
```